src/Paket/Paket.fsproj

```diff
@@ -30,6 +30,8 @@
     <StartArguments>update</StartArguments>
     <StartAction>Project</StartAction>
     <StartProgram>paket.exe</StartProgram>
+    <StartWorkingDirectory>C:\projects\SourceLink</StartWorkingDirectory>
```

Comment:
@latkin @enricosada I hope we can solve this issue with private debug info in F# vNext...